webrtc.html

```diff
@@ -4830,11 +4844,39 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         </section>
       </div>
       <div>
+        <pre class="idl">enum RTCDtxStatus {
+        "disabled",
+        "enabledIfNegotiated"
```

Comment:
I'm fine with this, but I thought Justin's suggestion was to just make this "enabled" so it was less verbose.  I'm fine either way.